src/components/AccountContainer.js

```diff
@@ -6,55 +6,41 @@ class AccountContainer extends Component {
 
   constructor() {
     super()
+    this.handleChange = this.handleChange.bind(this)
 
     // we have provided this default state for you,
     // use this to get the functionality working
     // and then replace the default transactions with a call to the API
 
     this.state = {
       searchTerm: '',
-      transactions: [
-        {
-          id: 1,
-          posted_at: "2017-02-28 11:00:00",
-          description: "Leather Pants, Gap co.",
-          category: "Fashion",
-          amount: -20000
-        },
-        {
-          id: 2,
-          posted_at: "2017-02-29 10:30:00",
-          description: "Paycheck from Bob's Burgers",
-          category: "Income",
-          amount: 100000
-        },
-        {
-          id: 3,
-          posted_at: "2017-05-24 10:53:00",
-          description: "'Pair Programming Illuminated' by Laurie Williams and Robert Kessler",
-          category: "Book",
-          amount: 1498
-        },
-        {
-          id: 4,
-          posted_at: "2017-05-24 08:52:00",
-          description: "Medium Iced Cold Brew, Gregory's Coffee",
-          category: "Coffee",
-          amount: 365
-        }
-      ]
+      transactions: []
     }
   }
 
+  componentDidMount() {
+    const URL = "https://boiling-brook-94902.herokuapp.com/transactions"
+    fetch(URL).then(res => res.json()).then(transactions => this.setState({transactions}))
+  }
+
+  searchTransactions(term) {
+   this.setState({
+     transactions: this.state.transactions.filter(transaction => transaction.category.includes(term) || transaction.description.includes(term))
```

Comment:
We discussed this in the review - by resetting the state, you're losing all the transactions that don't match. Instead, you can do this filtering but don't save it to state - just pass it to the component at that time.